tests/test_rpc.py

```diff
@@ -68,7 +67,7 @@ def test_listvaults(revaultd_manager, bitcoind):
     revaultd_manager.wait_for_log(f"Vault at .*{txid}.* is now confirmed")
     vault = revaultd_manager.rpc.call("listvaults")["vaults"][0]
     assert vault["status"] == "funded"
-    assert vault["updated_at"] > vault["received_at"]
+    assert vault["funded_at"] is not None
```

Comment:
Assert that the others are None

Reply:
Actually here you could assert that `funded_at` is the block timestamp of the last block

Reply:
Also, you need to add these checks after the restart (line ~80)